pages/api/question.ts

```diff
@@ -86,3 +92,13 @@ export type UnitPriceModel = {
   singularFruit: string,
   pluralFruit: string,
 };
+
+  export type SalesTaxModel = {
+    price: number;
+    number: number;
```

Comment:
is there a more descriptive property name than `number` that you could use?

Reply:
thanks.  I changed it to numberOfToys.